packages/terra-functional-testing/src/terra-cli/wdio/test-runner.js

```diff
@@ -97,7 +99,7 @@ class TestRunner {
       gridUrl,
       locales,
       themes,
-      ...launcherOptions // hostname, port, baseUrl, suite, spec, and keepAliveSeleniumDockerService
+      ...launcherOptions // hostname, port, baseUrl, suite, spec, updateScreenshots, and keepAliveSeleniumDockerService
```

Comment:
If possible, I think we should avoid flattening our custom configuration options onto the wdio config. A collision is probably unlikely, but I think we should have some distinction between our keys and WebDriverIO's keys similar to the `serviceOptions` key we added. Maybe `launcherOptions` or `customLauncherOptions` or `terraLauncherOptions`. What are your thoughts?

Reply:
Yeah, I think it does make sense to separate them out. Is this what you're thinking (also need to not blow away the global Terra object [here](https://github.com/cerner/terra-toolkit/blob/440f788697e86c4c4aca1630d134ea1942948a13/packages/terra-functional-testing/src/services/wdio-terra-service.js#L23)) or something more sophisticated?

![Screen Shot 2021-02-08 at 4 54 07 PM](https://user-images.githubusercontent.com/22618655/107291825-632f0d80-6a2e-11eb-9e47-737b33b7b19f.png)

Reply:
I was thinking of passing the options through onto the config, but scoped into a launcherOptions. The global wouldn't get defined here:

![image](https://user-images.githubusercontent.com/6720991/107297821-c4a8a980-6a39-11eb-8623-ccba7c12dea2.png)

This would add `launcherOptions` onto the `wdio.conf.js` but in a variable scoped to our custom options being passed through

Reply:
Gotcha. I tweaked things around just a bit: https://github.com/cerner/terra-toolkit/pull/568/commits/304bb561772b452a6d5949267e03938b35c0e64c